client/components/NewMessage.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+
+import people from '../people.js'
+
+const styles = {
+  borderRadius: '150px',
```

Comment:
You can also use `50%` here and then you won't have to change it if we decide to alter the width/height later.